### DISCUSSION TITLE
Beeps voice

### DISCUSSION THREAD

**Discussion by hilsonp** [pipeeeeees/waze-voicepack-links#15] [Ideas]
Hello,
Thank you for this project. 

Before I create one, maybe someone has made it already. 

I wish I could have a voice with short beeps. 

When I'm on journeys that I know, announcements are annoying/disruptive, especially while listening to podcasts. 

On the other hand, shutting down completely announcements is not good because I can miss a useful rerouting. 

I always wished I could select a beep only voice. 

Do you know such a voice?

Thank you.

**Comment by pipeeeeees** [comment]
I do not know of such a voice, but this is an interesting idea! As with all other packs, shoot me the mp3 files and I will make it shareable and of high audio quality.

**Comment by pipeeeeees** [comment]
Someone needed a pack made recently and wanted me to share this with you:  https://waze.com/ul?acvp=5a349e69-794e-417c-b3d1-96ab47597481

It's just beeps but for Mario

**Reply by hilsonp** [reply]
Thank you. I'll have a look !
